std/experimental/allocator/package.d

```diff
@@ -1601,6 +1601,125 @@ unittest //bugzilla 15721
 }
 
 /**
+Allocates a multidimensional array of elements of type T.
+
+Params:
+N = number of dimensions
```

Comment:
Thx for the good examples, though no exception is justified there. We should probably slowly fix those as the code gets touched instead of introducing more instances.

cc @wilzbach: all values and functions are `likeThis`, all types are `LikeThis`, aliases that may be either should generally be `likeThis`, modules and labels should be `like_this`.